### ISSUE TITLE
Text tool doesn't allow you to click inside to move the cursor or highlight text

### ISSUE THREAD

**Issue by bigbluebutton-issue-import** [bigbluebutton/bigbluebutton#2477]
Originally reported on Google Code with ID 1725

```
What steps will reproduce the problem?
1. Create a text annotation
2. Type some text in
3. Click to somewhere in the message

Expected:
The cursor should move to where you clicked

Actual:
The text tool is closed and the annotation is sent.

```

Reported by `capilkey` on 2014-02-06 23:17:36

**Comment by pedrobmarin** [comment]
Still a valid issue. I'm migrating this to the HTML5 Client.

**Comment by antobinary** [comment]
Resolved in ~BBB 2.6.0-beta.5
